scripts/step03_explore_examples/02_advanced_rag/src/benchmark.py

```diff
@@ -0,0 +1,163 @@
+import logging
+import textwrap
+import sys
+import re
+from tabulate import tabulate
+from pydantic import BaseModel
+
+from gen_ai_hub.proxy.langchain.openai import ChatOpenAI
+from gen_ai_hub.proxy.core.proxy_clients import get_proxy_client
+
+from langchain.chains import RetrievalQA
+from langchain.prompts import PromptTemplate
+
+from helpers.config import TERRAFORM_TABLE_NAME, JUDGE_LLM
+from helpers.factory import setup_components
+
+log = logging.getLogger(__name__)
+
+class ValidationResult(BaseModel):
+    query: str
+    actual_answer: str
+    rating: float
+    reasoning: str
+
+    class Config:
```

Comment:
poetry lock refers to Pydantic V2, where usage of Config is obsolete: 

name = "pydantic"
**version = "2.8.2"**

https://docs.pydantic.dev/2.2/migration/#changes-to-config